dipy/reconst/csdeconv.py

```diff
@@ -964,7 +964,7 @@ def recursive_response(gtab, data, mask=None, sh_order=8, peak_thr=0.01,
     where_dwi = lazy_index(~gtab.b0s_mask)
     response_p = np.ones(len(n))
 
-    for num_it in range(1, iter):
+    for num_it in range(iter):
```

Comment:
While you're at it, would you mind changing the name of this variable? `iter` is a key-word (a function) in the language.